### PR TITLE
Open janus ports by default

### PR DESCRIPTION
**Pull request by njam**

### CODE REVIEW

**Comment by njam** [comment]
@kris-lab @tomaszdurka please review

File: modules/janus/templates/config [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [general]
-interface = <%= @bind_address %>	; Interface to use (will be used in SDP)
+<% if !@bind_address %>;<% end %>interface = <%= @bind_address %>	; Interface to use (will be used in SDP)
```
Comment (tomaszdurka) [inline modules/janus/templates/config]: Is this valid? I thought you are supposed to use unless :panda_face:

Comment (tomaszdurka) [inline modules/janus/templates/config]: Or `if @bind_address.nil?` ?
I really thought this falsy value doesn't work reliably in ruby.

**Comment by tomaszdurka** [comment]
Rest looks good.

**Comment by njam** [comment]
retest this please

**Pull request merged**